vendor/bat-native-ledger/src/bat/ledger/internal/legacy/media/media.cc

```diff
@@ -22,8 +22,8 @@ bool HandledByGreaselion(const std::string media_type) {
   return false;
 #else
   return media_type == "github" || media_type == "reddit" ||
-         media_type == "twitter" || media_type == "vimeo" ||
-         media_type == "youtube";
+         media_type == "twitch" || media_type == "twitter" ||
+         media_type == "vimeo" || media_type == "youtube";
```

Comment:
I _think_ this can just return true for desktop, now that all of the supported publishers use Greaselion.